R/use_tic.R

```diff
@@ -97,7 +97,10 @@ use_tic <- function(quiet = FALSE) {
   #' 1. Create a default `appveyor.yml` file
   #'    (depending on repo type, overwrite after confirmation
   #'    in interactive mode only)
-  if (needs_appveyor(repo_type)) use_appveyor_yml()
+  if (needs_appveyor(repo_type)) {
```

Comment:
Should we use `needs_circle()` instead to determine if we add a yml?

Reply:
Thanks. Don't know why I put it within the appveyor if clause.